README.md

```diff
@@ -1,23 +1,31 @@
-# Dogecoin Core [DOGE, Ð]
-==========================
+<p align="center">
+<img src="http://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png"
+</p>
+<h1 align="center">Dogecoin Core [DOGE, Ð]</h1>
```

Comment:
Better to keep the markdown, as that's readable in text editors.

Reply:
Noted. Will revert back to markdown in next commit.